packages/react/src/components/Nav/__snapshots__/Nav.test.tsx.snap

```diff
@@ -111,7 +111,22 @@ exports[`Nav render Nav with overrides correctly 1`] = `
           >
             
           </i>
-          Group
+          <div
```

Comment:
Was this change intentional?

Reply:
Yes. Group headers misaligned visually with all other content. Since it was by 2px it did seem unintended. Also, since the `linkText` style doesn't enforce font size applying that same style works nicely and will allow for smoother style overrides.